.github/workflows/stale.yml

```diff
@@ -0,0 +1,16 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
```

Comment:
Not sure we need this to run daily - but for now I think this is ok 👍

Reply:
If it doesn't run daily, then there's a chance the stale PR isn't marked stale when it should be